.travis.yml

```diff
@@ -15,11 +15,65 @@ matrix:
         - npm run linter
 
     - language: python
+      
       python:
         - "3.6"
+      sudo: required
+
+      services:
+        - docker
+
+      before_install:
+        - python --version
+        - uname -a
+        - lsb_release -a
+        - docker --version
+        - docker-compose --version
+        # shutdown postgres because sometimes is is alrady up ???
+        - sudo service postgresql stop
+        # wait for postgresql to shutdown
+        - while sudo lsof -Pi :5432 -sTCP:LISTEN -t; do sleep 1; done
+
       install:
         - pip3 install pylint
+        - pip3 install pytest-cov
+        - pip3 install coveralls
+        - pip3 install -r modules/pytest_docker/requirements.txt
+        - pip3 install -r modules/s3wrapper/requirements.txt
+        - pip3 install -r modules/simcore_sdk/models/requirements.txt
+        - pip3 --version
+      
+      # Todo MaG: I have to move everything into one docker-compose.yml due to conftest.py issues
       before_script:
         - pylint --version
+        # change python path for 2nd party libraries
+        - export PYTHONPATH=${PYTHONPATH}:${TRAVIS_BUILD_DIR}/modules/s3wrapper/src
+        # pre-cache docker images to avoid time-outs later on while the tests turn
+        - docker-compose -f modules/pytest_docker/tests/docker-compose.yml pull
```

Comment:
Why pull and then build and not just build the image??

Reply:
Hi Pedro
Thanks for the suggestions. Whats wrong with the linting? Am I not using the correct configuration? I get a straight 10.0...